scripts/install-latest-ci.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -o errexit -o nounset -o pipefail
+set -o xtrace
+
+: ${OWNER:=rancher-sandbox}
+: ${REPO:=rancher-desktop}
+: ${BRANCH:=main}
+: ${WORKFLOW:=Package}
+: ${RESULTS:=30}
+
+# Get a list of all successful workflow runs for this repo.
+# I couldn't find a way to filter by workflow name, so get the last $RESULTS
+# runs and hope that at least one of them was for $WORKFLOW.
+API="repos/$OWNER/$REPO/actions/runs?branch=$BRANCH&status=success&per_page=$RESULTS"
+FILTER="[.workflow_runs[] | select(.name == \"$WORKFLOW\")] | .[0].id"
+
+ID=$(gh api "$API" --jq "$FILTER")
+if [ -z "$ID" ]; then
+    echo "No successful $WORKFLOW run found for branch $BRANCH"
+    exit 1
+fi
+
+download_artifact() {
+    FILENAME=$1
+
+    # Get the artifact id for the package
+    API="repos/$OWNER/$REPO/actions/runs/$ID/artifacts"
+    FILTER=".artifacts[] | select(.name == \"$FILENAME\").id"
+
+    ARTIFACT_ID=$(gh api "$API" --jq "$FILTER")
+    if [ -z "$ARTIFACT_ID" ]; then
+        echo "No download url for '$FILENAME' from $WORKFLOW run $ID"
+        exit 1
+    fi
+
+    # Download the package. It requires authentication, so use gh instead of curl.
+    API=repos/$OWNER/$REPO/actions/artifacts/$ARTIFACT_ID/zip
```

Comment:
Shouldn't API be quoted?

Reply:
It doesn't really matter; `shellcheck` would complain if it was necessary.

So it really is just a matter of aesthetics. However, I agree that it would look better quoted, so will change it. 😄